### PR TITLE
[RA1 all Ch] Proofreading

### PR DESCRIPTION
**Pull request by karinesevilla**
Proofreading for all Chapters

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 1]
```diff
@@ -187,19 +187,16 @@ other. In case of CaaS some cloud infrastructure features (e.g.: HW
 management or multitenancy) are implemented by using an underlying
 **IaaS** layer.
 
-**Cloud Infrastructure Hardware Profile:** defines the behaviour,
+**Cloud Infrastructure Hardware Profile:** Defines the behaviour,
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:190]: we do not need to capitalize is the term is part of the sentence

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 2]
```diff
@@ -274,11 +274,11 @@ reduces the virtual to physical pages’ address translations. Without
 huge pages enabled high TLB miss rates would occur thereby degrading
 performance.
 
-**Hypervisor:** a software that abstracts and isolates workloads with
+**Hypervisor:** A software that abstracts and isolates workloads with
 their own operating systems from the underlying physical resources. Also
 known as a virtual machine monitor (VMM).
 
-**Instance:** is a virtual compute resource, in a known state such as
+**Instance:** Is a virtual compute resource, in a known state such as
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:281]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 3]
```diff
@@ -274,11 +274,11 @@ reduces the virtual to physical pages’ address translations. Without
 huge pages enabled high TLB miss rates would occur thereby degrading
 performance.
 
-**Hypervisor:** a software that abstracts and isolates workloads with
+**Hypervisor:** A software that abstracts and isolates workloads with
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:277]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 4]
```diff
@@ -187,19 +187,16 @@ other. In case of CaaS some cloud infrastructure features (e.g.: HW
...
-**Host Profile:** is another term for a Cloud Infrastructure Hardware
-Profile.
-
 **Cloud Infrastructure Profile:** The combination of the Cloud
 Infrastructure Software Profile and the Cloud Infrastructure Hardware
 Profile that defines the capabilities and configuration of the Cloud
 Infrastructure resources available for the workloads.
 
-**Cloud Infrastructure Software Profile:** defines the behaviour,
+**Cloud Infrastructure Software Profile:** Defines the behaviour,
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:199]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 5]
```diff
@@ -293,11 +293,11 @@ Infrastructure. Note, as with all capabilities, Monitoring may be
 unavailable or intentionally disabled for security reasons in a given
 cloud infrastructure instance.
 
-**Multi-tenancy:** feature where physical, virtual or service resources
+**Multi-tenancy:** Feature where physical, virtual or service resources
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:296]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 6]
```diff
@@ -321,7 +321,7 @@ infrastructure. **NFVI** and **VNF** are the top-level conceptual
 entities in the scope of Network Function Virtualisation. All other
 components are sub-entities of these two main entities.
 
-**Network Service (NS):** composition of **Network Function**\ (s)
+**Network Service (NS):** Composition of **Network Function**\ (s)
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:324]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 7]
```diff
@@ -377,13 +377,14 @@ hardware. The storage access software provides data request interfaces
 (APIs) and the SDS controller software provides storage access services
 and networking.
 
-**Tenant:** cloud service users sharing access to a set of physical and
+**Tenant:** Cloud service users sharing access to a set of physical and
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:380]: please see capitalization rule

File: doc/ref_arch/openstack/chapters/chapter04.rst [hunk 8]
```diff
@@ -1762,5 +1762,5 @@ number of open-source tools are available for this purpose including:
 -  Triple-O :cite:p:`tripl`: for
    installing, upgrading and operating OpenStack clouds
 
-These installers are described in more details in
-:ref:`chapters/chapter07:operations and life cycle management`.
+These installers are described in more details
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter04.rst:1765]:
```suggestion
These installers are described in more detail
```

**Review by pgoyal01 — CHANGES_REQUESTED** [review]

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst:190]: @pgoyal01 Okay, I will modify and apply ths rule

Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: @pgoyal01 Done!

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 2 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 3 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 4 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 5 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 6 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 7 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Done

File: doc/ref_arch/openstack/chapters/chapter04.rst (hunk 8 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter04.rst]: Done

**Comment by karinesevilla** [comment]
@pgoyal01 Corrections made, please review again

**Review by ildikov — APPROVED** [review]
Thanks Karine!

**Pull request merged**